src/test/scala/com/cognite/spark/datasource/AssetsRelationTest.scala

```diff
@@ -220,41 +208,33 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
 
     // Check if post worked
     val assetsFromTestDf = retryWhile[Array[Row]](
-      spark.sql(s"select * from destinationAssets where source = '$source'").collect,
+      spark.sql(s"select * from destinationAssets where source = '$source' and description = 'foo'").collect,
       df => df.length < 100)
     assert(assetsFromTestDf.length == 100)
 
-    val description = "spark-assets-upsert-testing-description"
-
     // Upsert assets
     spark
       .sql(s"""
-                 |select id,
-                 |path,
-                 |depth,
-                 |name,
-                 |parentId,
-                 |types,
-                 |'bar' as description,
-                 |map("foo", null, "bar", "test") as metadata,
-                 |source,
-                 |sourceId,
-                 |createdTime,
-                 |lastUpdatedTime
-                 |from destinationAssets
-                 |where source = '$source'""".stripMargin)
+              |select externalId,
```

Comment:
Are we intentionally changing the order here because we have to? Maybe having `id` first is the better order? We can change this in the SDK to make it "the most natural". What do you think, @hakontro and @Tapped?

Reply:
Jetfire have stopped using the `insertable` functionality of relations for all resource types except Datapoints. I think we should stop having insertable relations, and instead only support save mode.

Reply:
No conscious decision on the order for these tests, the correct order is ensured by `.select(destinationDf.columns.map(col): _*)` anyways.

Reply:
@Tapped we need `insertable` for writing with SparkSQL, I believe?

Reply:
Yeah, you'll need that but it isn't a pleasant experience, since read and write schema is in practice different + you've to make the order of the columns correct.

Reply:
But in order to use `insert into` with Spark SQL, this is what's being used. We have no choice but to support it.

So the question is what order makes sense for the user. Probably best to match the order in the documentation? I think I tried to do that with the Scala SDK but we should double check it.